### PR TITLE
render an extended placeholder for head and footer

### PR DESCRIPTION
**Pull request by kalbasit**
I'd like to add custom javascript in the head and footer sections, this allows me to add those Javascript files in the custom head/footer partials in my site.

### CODE REVIEW

**Comment by kalbasit** [comment]
@panr ping

File: exampleSite/layouts/partials/custom_footer.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<script type="text/javascript">
+	console.log("Custom Footer is on!");
```
Comment (panr) [inline exampleSite/layouts/partials/custom_footer.html]: Do we need this here? 🤔

File: exampleSite/layouts/partials/custom_head.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<script type="text/javascript">
+	console.log("Custom Head is on!");
```
Comment (panr) [inline exampleSite/layouts/partials/custom_head.html]: As above

File: layouts/partials/custom_footer.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<!--
+To add custom footer section, please create
+`layouts/partials/custom_footer.html` in your Hugo directory.
```
Comment (panr) [inline layouts/partials/custom_footer.html]: I'm thinking about the name... Is it `custom_footer` or just `extended_footer`? Maybe it's not a big deal, but in this case we don't override whole footer, but only extend it 🤔

File: layouts/partials/custom_head.html [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<!--
+To add custom head section, please create `layouts/partials/custom_head.html` in your
```
Comment (panr) [inline layouts/partials/custom_head.html]: As above (filename)

File: exampleSite/layouts/partials/custom_footer.html (hunk 1 above)
Comment (kalbasit) [reply exampleSite/layouts/partials/custom_footer.html]: We don't really, it's more of a POC that it works. I can remove them.

File: layouts/partials/custom_footer.html (hunk 3 above)
Comment (kalbasit) [reply layouts/partials/custom_footer.html]: That's a good point. I think `extended_footer` makes more sense, I'll update the PR.

File: exampleSite/layouts/partials/custom_footer.html (hunk 1 above)
Comment (kalbasit) [reply exampleSite/layouts/partials/custom_footer.html]: I removed this file.

File: exampleSite/layouts/partials/custom_head.html (hunk 2 above)
Comment (kalbasit) [reply exampleSite/layouts/partials/custom_head.html]: I removed this file.

File: layouts/partials/custom_footer.html (hunk 3 above)
Comment (kalbasit) [reply layouts/partials/custom_footer.html]: I renamed it to `extended_footer`.

File: layouts/partials/custom_head.html (hunk 4 above)
Comment (kalbasit) [reply layouts/partials/custom_head.html]: I renamed it to `extended_head`.

**Comment by kalbasit** [comment]
@panr I've updated the PR and addressed your comments, PTAL. I have not paid specific attention to my commit messages, so please squash & merge!

**Comment by panr** [comment]
Looks good! Thanks! 🎉

**Pull request merged**